### PR TITLE
Loulou

### PR DESCRIPTION
**Pull request by ellpei**
Using props now, lots of changes

### CODE REVIEW

File: src/components/DeltaButton.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <span id="likes">0</span>
+  </span>
+</template>
+<script>
+export default {
+  name: 'DeltaButton',
+  props: ['deltas'],
+  methods: {
+    Like: function () {
+      document.getElementById('likes').innerHTML = parseInt(document.getElementById('likes').innerHTML) + 1
```
[the remark this answers is not in the document]
Comment (ellpei) [reply src/components/DeltaButton.vue]: Detta funkar inte: 
<template>
  <span>
    <span id="likeButton" v-on:click="Like()">delta</span>
    <span id="likes">{{likes}}</span>
  </span>
</template>
<script>
export default {
  name: 'DeltaButton',
  data () {
    likes: 0
  },
  props: ['deltas'],
  methods: {
    Like: () => this.state.likes++
  }
}

File: src/components/InfoBubble.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    </div>
+  </div>
+  </template>
+<script>
+export default {
+  name: 'MakePost',
+  props: ['message'],
+  methods: {
+    close: function () {
+      document.getElementById('parent').removeChild(document.getElementById('box'))
```
[the remark this answers is not in the document]
Comment (ellpei) [reply src/components/InfoBubble.vue]: Nu: 

'<template>
  <div id="parent">
    <div id="box" v-if="visible">
      <div id="closeBtn" v-on:click="close()">X</div>
      <p>{{message}}</p>
    </div>
  </div>
  </template>
<script>
export default {
  name: 'MakePost',
  props: ['message', 'visible'],
  methods: {
    close: function () {
      this.visible = false
    }
  }
}
</script>' 

Den klagar dock fortfarande

File: src/components/DeltaButton.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (ellpei) [reply src/components/DeltaButton.vue]:
```<template>
  <span>
    <span id="likeButton" v-on:click="Like()"><img src="@/assets/deltaSVG.svg" alt="delta" id="delta" width="20px"/></span>
    <span id="likes">{{likes}}</span>
  </span>
</template>
<script>
export default {
  name: 'DeltaButton',
  data () {
    return {
      likes: 0
    }
  },
  props: ['deltas', 'postId', 'userID'],
  created: function () {
    //  this doesn't work??? likes = deltas
  },
  methods: {
    Like: function () {
      this.likes++
      //  update database using id prop
    }
  }
}
</script>```

File: src/components/DeltaButton.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export default {
+  name: 'DeltaButton',
+  data () {
+    return {
+      likes: 0
+    }
+  },
+  props: ['deltas', 'postId', 'userID'],
+  created: function () {
+    //  this doesn't work??? likes = deltas
```
Comment (Pranz) [inline src/components/DeltaButton.vue]: you can do
```
data() {
  return {
    likes: this.deltas
  }
}
```
but this is not recommended. If the prop changes, then this component will be rerendered and the new state will be added. However you can remove the state from here and make the parent change it state, only passing down the state via props.

**Comment by Pranz** [comment]
I'm leaving this open if you want to make a change to the deltabutton but feel free to merge in otherwise.

**Pull request merged**